### ISSUE TITLE
app.json should use "heroku-redis" in addition to "redistogo"

### ISSUE THREAD

**Issue by dzuelke** [opentok/servicekit-php#8]
Offers more storage :)

Maybe there could be multiple chained `getenv` calls in the config like so:

```
$url = getenv('REDIS_URL', getenv('REDISTOGO_URL', getenv('REDISCLOUD_URL')));
```

That way, it tries all possible providers.

**Comment by aoberoi** [comment]
@dzuelke good idea. in fact, if it offers more storage, i think it should be the preferred provider. also, we would want to update the `app.json` file to reflect that, and the `README.md` Appendix copy to mention that you can choose either provider.

would you be willing to send a PR for this change?

**Comment by dzuelke** [comment]
Can do, @aoberoi !

**Comment by dzuelke** [comment]
See #10, @aoberoi !
